docs/releasing/publishing.md

```diff
@@ -65,43 +65,18 @@ Developers should pair on releases. When remote working, it can be useful to be
 
 ## Publish a release to npm
 
-1. Check out the **main** branch and pull the latest changes.
-
-   If there was an interruption between the raising of the PR and its merge,
-   or it's another developer running the publication to npm, rebuild the package with:
-
-   ```shell
-   npm run build:package
-   ```
-
-2. Sign in to npm (`npm login`), using the credentials for the **govuk-patterns-and-tools** npm user from Bitwarden.
-
-3. Run `npm run publish-release`, which will prompt you to check whether the npm tag looks as expected.
+1. Once the release pull request has been merged, open the Actions tab on the `alphagov/govuk-frontend` repo.
 
-   If you're following these instructions, you're probably doing a sequential release, meaning
-   the tag should be 'latest'.
+2. Select the "RELEASE: Publish to npm" workflow and run the workflow on the `main` branch. This will publish the release to npm.
```

Comment:
Not for right now but it'd be good to link directly to the workflow once we know where it lives in github's UI

Reply:
Good point! It should be at https://github.com/alphagov/govuk-frontend/actions/workflows/publish-to-npm.yaml unless we change the name of the yaml file, so I'll add that in.

Reply:
Done!